bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -969,6 +969,48 @@ impl Connection {
         }
         Ok(secret)
     }
+
+    /// Retrieves the size of the serialized connection
+    pub fn serialization_length(&self) -> Result<usize, Error> {
+        unsafe {
+            let mut length = 0;
+            s2n_connection_serialization_length(self.connection.as_ptr(), &mut length)
+                .into_result()?;
+            Ok(length.try_into().unwrap())
+        }
+    }
+
+    /// Serializes the TLS connection into the provided buffer
+    pub fn serialize_connection(&self, output: &mut [u8]) -> Result<(), Error> {
```

Comment:
connection::serialize_connection / conn.serialize_connection seems repetitive. Same for deserialize
```suggestion
    /// Serializes the TLS connection into the provided buffer
    pub fn serialize(&self, output: &mut [u8]) -> Result<(), Error> {
```